lib/ord.js

```diff
@@ -24,10 +24,11 @@ const initializeAppConfig = (csn) => {
     const packageName = packageJson.name;
     const appName = packageJson.name.replace(/^[@]/, "").replace(/[@/]/g, "-");
     const modelKeys = Object.keys(csn.definitions);
+    const endpoints = new Set();
     const events = [];
     const serviceNames = [];
     const lastUpdate = getRFC3339Date();
-    let odmEntity = [];
+    let odmEntities = [];
```

Comment:
I would like use const if odmEntities would not be assigned to other values

Reply:
Done.